docs/secondary-network.md

```diff
@@ -0,0 +1,142 @@
+following:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-controller.conf: |
+    featureGates:
+      AntreaIPAM: true
+  antrea-agent.conf: |
+    featureGates:
+      AntreaIPAM: true
+      SecondaryNetwork: true
+```
+
+Antrea leverages the `NetworkAttachmentDefinition` CRD from [Kubernetes Network
+Plumbing Working Group](https://github.com/k8snetworkplumbingwg/multi-net-spec)
+to define secondary networks. You can import the CRD to your cluster using the
+following command:
+
+```bash
+kubectl apply -f https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/raw/master/artifacts/networks-crd.yaml
+```
+
+## Secondary OVS bridge configuration
+
+A VLAN secondary interface will be connected to a separate OVS bridge on the
+Node. You can specify the secondary OVS bridge configuration in the
+`antrea-agent` configuration, and `antrea-agent` will automatically craete the
+OVS bridge on based on the configuration. For example, the following
+configuration will create an OVS bridge named `br-secondary`, with a physical
+interface `eth1`.
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: antrea-config
+  namespace: kube-system
+data:
+  antrea-agent.conf: |
+    secondaryNetwork:
+    ovsBridges: [{"bridgeName": "br-secondary", "physicalInterfaces": ["eth1"]}]
+```
+
+At the moment, Antrea supports only a single OVS bridge for secondary networks,
+and supports only a single physical interface on the bridge. The physical
+interface cannot be the Node's management interface, otherwise a Node's
+management network connectivity can be broken after `antrea-agent` creates the
+OVS bridge and moves the management interface to the bridge.
+
+## Secondary VLAN network configuration
+
+A secondary VLAN network is defined by a NetworkAttachmentDefinition CR. For
+example, the following NetworkAttachmentDefinition defines a VLAN network
+`vlan100`.
+
+```yaml
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: vlan100
+spec:
+  config: '{
+      "cniVersion": "0.3.0",
+      "type": "antrea",
+      "networkType": "vlan",
+      "mtu": 1500,
+      "vlan": 100,
+      "ipam": {
+        "type": "antrea",
+        "ippools": ["vlan100-ipv4", "vlan100-ipv6"]
+      }
+    }'
+```
+
+`antrea-agent` will connect a Pod secondary interface of a VLAN network to the
+secondary OVS bridge on the Node. If a non-zero VLAN is speficied in the
+network's `config`, `antrea-agent` will configure the VLAN ID to the OVS port,
+so the interface's traffic will be isolated within the VLAN. And before the
+traffic is forwarded out the Node via the secondary bridge's physical interface,
+OVS will insert the VLAN tag to the packets.
+
+A few extra notes about the NetworkAttachmentDefinition `config` fields:
+
+* `type` - must be set to `antrea`.
+* `networkType` - the only supported network type is `VLAN` as of now.
```

Comment:
is it case sensitive? You use `vlan` in the examples

Reply:
`vlan` makes sense. Changed.